### ISSUE TITLE
Create "add from CSV" option to appropriate screens

### ISSUE THREAD

**Issue by winterweird** [winterweird/jpractice#36]
I think it would be extremely useful to have this functionality, especially considering how there is already a functionality for *exporting* to CSV. It would be nice to have this option both at the individual list level, where it cherry picks the entries which go into the same list, as well as on the list of lists level, where it inserts the elements in existing lists as appropriate, and otherwise creates new lists if the given lists do not already exist.

This functionality should handle duplicates and other potential errors gracefully. It would be nice, especially for debugging purposes, to in the end receive an overview of unresolved errors encountered while adding the CSV entries.

This needs to be done before #28 (search history) can be completed, because I want to be able to restore my database as completely and as immediately as possible when it's inevitably wiped due to being upgraded to make room for the new database features.
